algorithm/graph/graph.go

```diff
@@ -34,6 +34,12 @@ type Graph interface {
 	// Edges Returns edges.
 	Edges() []map[int]int
 
+	// Degree Returns degree of node.
+	Degree(index int) int
+
+	// Connected Returns degree of node.
```

Comment:
The comment should be 'Returns num Connected of node'.

Reply:
take and done